Libs/vtkITK/vtkITKArchetypeImageSeriesReader.h

```diff
@@ -111,24 +111,24 @@ class VTK_ITK_EXPORT vtkITKArchetypeImageSeriesReader : public vtkImageAlgorithm
   void SetDesiredCoordinateOrientationToAxial ()
   {
     this->DesiredCoordinateOrientation =
-      itk::SpatialOrientation::ITK_COORDINATE_ORIENTATION_RAI;
-    ///     itk::SpatialOrientation::ITK_COORDINATE_ORIENTATION_RPS;
+      itk::SpatialOrientationEnums::ValidCoordinateOrientations::ITK_COORDINATE_ORIENTATION_RAI;
```

Comment:
Since it was introduced above, looks like we should use `CoordinateOrientationCode` here. I will update this and include the change in a force push.